rpc/wrtc_client.go

```diff
@@ -230,37 +218,51 @@ func dialWebRTC(
 		}
 
 		var pendingCandidates sync.WaitGroup
+
+		// waitOneHost is closed when the first ICE candidate of type `Host` (e.g: 127.0.0.1) is
+		// found.
 		waitOneHost := make(chan struct{})
 		var waitOneHostOnce sync.Once
 		peerConn.OnICECandidate(func(icecandidate *webrtc.ICECandidate) {
 			if exchangeCtx.Err() != nil {
+				// We've decided to bail.
```

Comment:
```suggestion
				// Client has canceled the dial or a timeout has occurred.
```

Reply:
Done, but used "Caller" instead of "Client"